### PR TITLE
added long/lat fields and buttons for common event locations for biztech

### PR DESCRIPTION
**Pull request by dchen150**
- added longitude and latitude fields to new event form and edit event form
- modified the Yup requirements for long/lat as needed 
- added a few buttons for the top 5 most common event locations from a quick glance through UBC BizTech event locations
- added an "info" icon that explains to execs why we want long/lat and how to get those values from Google Maps. 

image

### CODE REVIEW

File: src/components/Forms/EditEvent.js [hunk 1]
```diff
@@ -118,6 +165,37 @@ export default function EditEventForm(props) {
...
+                            id="latitude"
+                            label="Latitude"
+                            fullWidth
+                            helperText={textFieldError("latitude")}
+                            error={!!textFieldError("latitude")}
+                            value={latitude}
+                            onChange={change.bind(null, "latitude")} />
+                        <InfoIcon onClick={handleInfoClick} style={{ cursor: 'pointer' }} />
+                    </Grid>
+                    <Grid item xs={12}>
```
Comment (ianmah) [inline src/components/Forms/EditEvent.js:189]: Let's make this above the lat/long stuff

File: src/components/Forms/EditEvent.js [hunk 2]
```diff
@@ -118,6 +165,37 @@ export default function EditEventForm(props) {
...
+                        <TextField
+                            id="longitude"
+                            label="Longitude"
+                            fullWidth
+                            helperText={textFieldError("longitude")}
+                            error={!!textFieldError("longitude")}
+                            value={longitude}
+                            onChange={change.bind(null, "longitude")} />
+                    </Grid>
+                    <Grid item xs={12} sm={4} style={{ display: 'flex' }}>
```
Comment (ianmah) [inline src/components/Forms/EditEvent.js:178]: why flex?

File: src/components/Forms/NewEvent.js [hunk 3]
```diff
@@ -8,11 +8,15 @@ import {
     KeyboardDateTimePicker
 } from '@material-ui/pickers';
 import ThemeProvider from '../ThemeProvider'
+import InfoIcon from '@material-ui/icons/Info'
+import Select from "@material-ui/core/Select";
+import MenuItem from '@material-ui/core/MenuItem';
+import { Typography } from '@material-ui/core';
```
Comment (ianmah) [inline src/components/Forms/NewEvent.js:14]:
```suggestion
import Typography from '@material-ui/core/Typography';
```

will allow for faster builds

File: src/components/Forms/NewEvent.js [hunk 4]
```diff
@@ -48,6 +52,50 @@ export default function NewEventForm(props) {
         return (errors[id] && submitCount > 0) || (touched[id] ? errors[id] : "")
     }
 
+    const handleInfoClick = () => {
+        window.alert(`Longitude and latitude are used for routing purposes for the mobile app. These values can be found on Google Maps by right clicking any location and pressing "What's here?"
```
Comment (ianmah) [inline src/components/Forms/NewEvent.js:56]: Don't think we've been including `window` when we use `alert`

```suggestion
        alert(`Longitude and latitude are used for routing purposes for the mobile app. These values can be found on Google Maps by right clicking any location and pressing "What's here?"
```

File: src/components/Forms/NewEvent.js [hunk 5]
```diff
@@ -115,16 +163,49 @@ export default function NewEventForm(props) {
...
+                        <TextField
+                            id="longitude"
+                            label="Longitude"
+                            fullWidth
+                            helperText={textFieldError("longitude")}
+                            error={!!textFieldError("longitude")}
+                            value={longitude}
+                            onChange={change.bind(null, "longitude")} />
+                    </Grid>
+                    <Grid item xs={12} sm={4} style={{ display: 'flex' }}>
```
Comment (ianmah) [inline src/components/Forms/NewEvent.js:187]: also why flex

File: src/components/Forms/NewEvent.js [hunk 6]
```diff
@@ -134,6 +215,9 @@ export default function NewEventForm(props) {
                             error={!!textFieldError("facebookUrl")}
                             onChange={change.bind(null, "facebookUrl")}
                         />
+                    </Grid>
+                    <Grid >
+
                     </Grid>
```
Comment (ianmah) [inline src/components/Forms/NewEvent.js:221]: ?

File: src/components/Forms/EditEvent.js (hunk 2 above)
Comment (dchen150) [reply src/components/Forms/EditEvent.js:178]: this is the result if i dont use flex
image

File: src/components/Forms/NewEvent.js (hunk 5 above)
Comment (dchen150) [reply src/components/Forms/NewEvent.js:187]: see above comment

**Pull request merged**